engine/access/state_stream/backend/backend_events_test.go

```diff
@@ -422,45 +443,35 @@ func (s *BackendEventsSuite) subscribe(
 }
 
 // requireEventsResponse ensures that the received event information matches the expected data.
-func (s *BackendEventsSuite) requireEventsResponse(v interface{}, expected *EventsResponse) {
-	actual, ok := v.(*EventsResponse)
-	require.True(s.T(), ok, "unexpected response type: %T", v)
-
+func (s *BackendEventsSuite) requireEventsResponse(actual *state_stream.EventsResponse, expected *state_stream.EventsResponse) {
 	assert.Equal(s.T(), expected.BlockID, actual.BlockID)
 	assert.Equal(s.T(), expected.Height, actual.Height)
 	assert.Equal(s.T(), expected.Events, actual.Events)
 	assert.Equal(s.T(), expected.BlockTimestamp, actual.BlockTimestamp)
 }
 
-// TestSubscribeEventsFromSporkRootBlock tests that events subscriptions starting from the spork
-// root block return an empty result for the root block.
+// TestSubscribeEventsFromSporkRootBlock verifies that when subscribing from the spork
+// root block, the stream starts from the first non-root block with actual events (no empty root response).
 func (s *BackendEventsSuite) TestSubscribeEventsFromSporkRootBlock() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
 	// setup the backend to have 1 available block
 	s.highestBlockHeader = s.blocks[0].ToHeader()
 
-	rootEventResponse := &EventsResponse{
-		BlockID:        s.rootBlock.ID(),
-		Height:         s.rootBlock.Height,
-		BlockTimestamp: time.UnixMilli(int64(s.rootBlock.Timestamp)).UTC(),
-	}
-
-	firstEventResponse := &EventsResponse{
+	firstEventResponse := &state_stream.EventsResponse{
 		BlockID:        s.blocks[0].ID(),
 		Height:         s.blocks[0].Height,
 		BlockTimestamp: time.UnixMilli(int64(s.blocks[0].Timestamp)).UTC(),
 		Events:         flow.EventsList(s.blockEvents[s.blocks[0].ID()]),
 	}
 
-	assertSubscriptionResponses := func(sub subscription.Subscription, cancel context.CancelFunc) {
-		// the first response should have details from the root block and no events
```

Comment:
this functionality changed? when streaming from the spork root block, the first result returned should be an empty response for the root block.